packages/caip/src/adapters/gatefi/index.ts

```diff
@@ -0,0 +1,42 @@
+
+import {
+  avalancheAssetId,
+  btcAssetId,
+  ethAssetId,
+  ltcAssetId
+} from '../../constants'
+
+const GateFiIdtoAssetIdmap = {
```

Comment:
```suggestion
const GateFiIdToAssetIdMap = {
```

can we be consistent with naming through this PR, please?